### DISCUSSION TITLE
Q Badger meter analogic counter

### DISCUSSION THREAD

**Discussion by alray31** [jomjol/AI-on-the-edge-device#2648] [Q&A]
Hi,

Im just wondring if my Badger Recordall water meter would work correctly with this. I saw a youtube video (link below) of someone configuring ai-on-the-edge for this exact same meter but he didnt define any ROI for the analogic counter. Can the analogic counter be read even of the ROI of it would basically be the whole meter area? Im looking to be able to read 279,171 m3, not only 279.1. would it be possible?
![20231005_080742](https://github.com/jomjol/AI-on-the-edge-device/assets/115857857/f5821304-d850-4bbc-9da8-7d354ec8fa80)
https://youtu.be/Uap_6bwtILQ?si=RYcboZAsC6GmnF8V

**Comment by caco3** [comment]
I think it should work -> just give it a try. in the worst case you will miss the 0.01 info. But the device will anyway be able to interpolate, so even with just the digits you should get a resolution below 0.1

**Reply by alray31** [reply]
Thank you for taking the time to answer!

I'll definitively try! I'm just curious to know how it can interpolate, is it like using the position of the last digit to interpolate, I mean it know that it's about  000.15 when it see the 0.1 is almost showing 0.2 or something like that?

**Reply by caco3** [reply]
I am not sure how it works on the digits. On the analog pointers it calculates the angle of the pointer, so it very easy to gain higher accuracy.
